corporate/tests/test_stripe.py

```diff
@@ -328,8 +335,7 @@ def test_initial_upgrade(self, *mocks: Mock) -> None:
         stripe_subscription = extract_current_subscription(stripe_customer)
         self.assertEqual(stripe_subscription.billing, 'charge_automatically')
         self.assertEqual(stripe_subscription.days_until_due, None)
-        self.assertEqual(stripe_subscription.plan.id,
-                         Plan.objects.get(nickname=Plan.CLOUD_ANNUAL).stripe_plan_id)
+        self.assertEqual(stripe_subscription.plan.nickname, Plan.CLOUD_ANNUAL)
```

Comment:
@rishig We create two plans in setUpClass. The monthly plan gets the ID `plan_NORMALIZED0001` while annual plans gets `plan_NORMALIZED0002`. These values are stored in Plan objects. In `test_initial_upgrade` test while normalizing the Customer fixture the annual plan gets the id `plan_NORMALIZED0001` since there is no other fixture. THis creates a mismatch.

Alternative slution to this is to dnt do normalization for `plan_` or do something more complex.